### ISSUE TITLE
Please help me

### ISSUE THREAD

**Issue by rafaepires** [unit8co/darts#1812]
Please help me I'm having problems with the tool.
I'm training the model using RNNModel using yahoo finance only the close values.
The loss values are: train_loss=0.000101, val_loss=0.00272

But when making the prediction (model.predict) the forecasting values are very bad.

I know this is just for reporting bugs but I don't know where to turn.

Please could someone help me?

**Comment by alexcolpitts96** [comment]
Without code, I can't be of much help; however, this looks like a classic case of overfitting. Try turning down the number of epochs or adding an early stopping callback.
